cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -91,6 +91,14 @@ extern "C" {
 #define STM32_PM_STANDBY      (0U)
 /** @} */
 
+#ifndef PM_BLOCKER_INITIAL
+/**
+ * @brief   Default initial PM blocker
+ * @todo    All modes are blocked per default, until PM is cleanly implemented
+ */
+#define PM_BLOCKER_INITIAL  { .val_u32 = 0x01010101 }
```

Comment:
this is now identical as in ```sys/pm_layered/pm.c```. Should we just drop it?

Reply:
Yes that was the alternative solution I proposed in the initial comment of this PR. If it's ok for you I can drop it no problem :)

Reply:
Ah, I just saw that. Yeah, let's drop it.

Reply:
Done